tests/apps/loqus/test_loqusdbapi.py

```diff
@@ -0,0 +1,56 @@
+import subprocess
+
+from cg.apps.loqus import LoqusdbAPI
+
+def test_instatiate(loqus_config):
+    """Test to instantiate a loqusdb api"""
+    ## GIVEN a loqusdb api with some configs
+
+    ## WHEN instantiating a loqusdb api
+    loqusdb = LoqusdbAPI(loqus_config)
+
+    ## THEN assert that the adapter was properly instantiated
+    assert loqusdb.uri == loqus_config['loqusdb']['database']
+    assert loqusdb.db_name == loqus_config['loqusdb']['database_name']
+    assert loqusdb.loqusdb_binary == loqus_config['loqusdb']['binary']
+
+def test_get_case(loqusdbapi, mocker):
+    """Test to get a case via the api"""
+    ## GIVEN a loqusdb api
+    case_id = 'a_case'
+    ## WHEN fetching a case with the adapter
+    mocker.patch.object(subprocess, 'check_output')
+    subprocess.check_output.return_value = b'[{"_id": "a_case"}]'
+    case_obj = loqusdbapi.get_case(case_id)
+    ## THEN assert that the correct case id is returned
+    assert case_obj['_id'] == case_id
+
+def test_get_case_non_existing(loqusdbapi, mocker):
+    """Test to get a case via the api"""
+    ## GIVEN a loqusdb api
+    case_id = 'a_case'
+    ## WHEN fetching a case with the adapter
+    mocker.patch.object(subprocess, 'check_output')
+    subprocess.check_output.side_effect = subprocess.CalledProcessError(1,'error')
+    case_obj = loqusdbapi.get_case(case_id)
+
+    ## THEN assert None is returned
+    assert case_obj == None
+
+def test_load(loqusdbapi, mocker, loqusdb_output):
+    """Test to load a case"""
+    ## GIVEN a loqusdb api and some info about a case
+    family_id = 'test'
+    ped_path = 'a ped path'
+    vcf_path = 'a vcf path'
+
+    ## WHEN loading the case to loqus with the api
+    mocker.patch.object(subprocess, 'check_output')
+    subprocess.check_output.return_value = loqusdb_output
+
+    data = loqusdbapi.load(family_id, ped_path, vcf_path)
+
+    ## THEN assert that the number of variants is 15
```

Comment:
Ok, it is just something that's printed in the log output that is now a fixture

Reply:
sure and if the comment is not there to explain something then I guess it could be removed altogether. 

Now we have: 
# GIVEN a loqusdb api and some info about a case
# WHEN loading the case to loqus with the api
# THEN assert that the number of variants is 15

I suggest the Given/When/Then is written so that it explains in english why the test makes sense, something like:

# GIVEN we have an inflated ball
# WHEN we apply a needle to it
# THEN the ball should become flat

In the above case, maybe the when clause could be written something like:
# WHEN  uploading a case with 15 variants to loqusdb
Does it make sense?

Reply:
See suggestion below